config/services.yaml

```diff
@@ -48,6 +48,15 @@ parameters:
     # it accepts any values allowed for X-Frame-Options
     security.x_frame_options: SAMEORIGIN
 
+
+    #container.dumper.inline_class_loader: true
+
+    # all paths below are resolved as assets (in /web)
```

Comment:
in `web`?

Reply:
fixed in 27a361432cccea4b9ad72371414a893de64b079e